js/components/frame.js

```diff
@@ -54,12 +54,6 @@ class Frame extends ImmutableComponent {
 
   componentDidMount () {
     this.updateWebview()
-    // forward postMessage events from webview to webContents
-    window.addEventListener('message', function (event) {
```

Comment:
This probably broke the preferences. If we are going to keep using contentWindow for them we should lock down the origin for it